automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala

```diff
@@ -68,14 +73,41 @@ trait UserFixtures extends CleanUp with ScaledTimeSpans with Eventually { self:
                                                 (implicit webDriver: WebDriver): Unit = {
 
     logger.info(s"withSignIn (${user.email}) starting ...")
+
     executeTestCodeWithSignIn(user, {
       // workaround for failed forceSignedIn
       var counter = 0
       retry(Seq.fill(2)(1.seconds)) ({
-        logger.info(s"withSignIn (${user.email}) opening SignInPage ...")
-        new SignInPage(FireCloudConfig.FireCloud.baseUrl).open
+        logger.info(s"withSignIn (${user.email}) opening SignInPage.")
+        try {
+          new SignInPage(FireCloudConfig.FireCloud.baseUrl).open
+        } catch {
+            case _: TimeoutException =>
+              logger.info("Encountered Selenium.TimeoutException. webDriver forcing page refresh.")
+              webDriver.navigate().refresh()
+        }
+
+        // take screenshot and log console text on Login page
```

Comment:
in addition to extra logs, take screenshot and console text before execute JS script.